services/system_delete.proto

```diff
@@ -30,28 +48,51 @@ import "basic_types.proto";
 import "timestamp.proto";
 
 /**
- * Delete a file or smart contract - can only be done with a Hedera administrative multisignature.
- * When it is deleted, it immediately disappears from the system as seen by the user, but is still
- * stored internally until the expiration time, at which time it is truly and permanently deleted.
- * Until that time, it can be undeleted by the Hedera administrative multisignature. When a smart
- * contract is deleted, the cryptocurrency account within it continues to exist, and is not affected
- * by the expiration time here. 
+ * Delete a file or contract bytecode as an administrative transaction.
```

Comment:
I don't understand the use case for this transaction type. It sounds like this allows privileged users to delete any non-system file. Is that correct?

Reply:
Primarily to allow for the council to remove a file from state if needed for some reason.  I can think of a few _possible_ scenarios, but cannot say those are the actual reasons.